streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/TaskAssignorConvergenceTest.java

```diff
@@ -623,11 +622,10 @@ private static void testForConvergence(final Harness harness,
         }
 
         if (rebalancePending) {
-            final StringBuilder message =
-                new StringBuilder().append("Rebalances have not converged after iteration cutoff: ")
-                                   .append(iterationLimit)
-                                   .append(harness.history);
-            fail(message.toString());
+            final String message = "Rebalances have not converged after iteration cutoff: " +
+                    iterationLimit +
+                    harness.history;
+            fail(message);
```

Comment:
If we move off `StringBuilder` than we can get rid of `String messsage`, too, and put the string directly into `fail(...)`

And we can change the formatting...